menderqa_pipeline.go

```diff
@@ -351,3 +355,57 @@ func stopBuildsOfStalePRs(log *logrus.Entry, pr *github.PullRequestEvent, conf *
 	return nil
 
 }
+
+func deleteStaleGitlabPRBranch(log *logrus.Entry, pr *github.PullRequestEvent, conf *config) error {
```

Comment:
This has nothing to do with `menderqa_pipeline`, it should be moved to `pullreq_pipeline`